src/bindings/cpp/tests/tests.hpp

```diff
@@ -24,15 +24,8 @@
 using namespace std;
 using namespace kdb;
 
-#define succeed_if(x, y)                                                                                                                   \
-	do                                                                                                                                 \
-	{                                                                                                                                  \
-		ASSERT_TRUE (x) << y;                                                                                                      \
-	} while (0)
-#define exit_if_fail(x, y)                                                                                                                 \
-	do                                                                                                                                 \
-	{                                                                                                                                  \
-		ASSERT_TRUE (x) << y;                                                                                                      \
-	} while (0)
+#define succeed_if(x, y) ASSERT_TRUE (x) << y;
```

Comment:
This is not a good change, see https://stackoverflow.com/questions/1067226/c-multi-line-macro-do-while0-vs-scope-block

Reply:
Oh, I forgot to remove the trailing semicolons. This should be fixed in the latest version of the PR.